variant_remapping_tools/tests/test_reads_to_remapped_variants.py

```diff
@@ -0,0 +1,70 @@
+from unittest import TestCase
+from variant_remapping_tools.reads_to_remapped_variants import fetch_bases, calculate_new_alleles, process_bam_file
+
+
+class TestProcess(TestCase):
+
+    def test_fetch_old_bases(self):
+        fasta_path = '../../tests/resources/genome.fa'
+        assert fetch_bases(fasta=fasta_path, contig='chr1', length=2, start=1) == 'CA'
+        assert fetch_bases(fasta=fasta_path, contig='chr1', length=1, start=48) == 'C'
+        assert fetch_bases(fasta=fasta_path, contig='chr1', length=1, start=98) == 'C'
+        assert fetch_bases(fasta=fasta_path, contig='chr1', length=3, start=353) == 'GGA'
+        assert fetch_bases(fasta=fasta_path, contig='chr1', length=1, start=1078) == 'G'
+        assert fetch_bases(fasta=fasta_path, contig='chr1', length=1, start=1404) == 'G'
+        assert fetch_bases(fasta=fasta_path, contig='chr1', length=3, start=1818) == 'AAC'
+        assert fetch_bases(fasta=fasta_path, contig='chr1', length=1, start=2030) == 'A'
+
+    def test_fetch_bases(self):
+        fasta_path = '../../tests/resources/new_genome.fa'
+        assert fetch_bases(fasta=fasta_path, contig='chr2', length=2, start=1) == 'CA'
+        assert fetch_bases(fasta=fasta_path, contig='chr2', length=1, start=98) == 'C'
+        assert fetch_bases(fasta=fasta_path, contig='chr2', length=1, start=353) == 'G'
+        assert fetch_bases(fasta=fasta_path, contig='chr2', length=1, start=1078) == 'A'
+        assert fetch_bases(fasta=fasta_path, contig='chr2', length=1, start=1404) == 'G'
+        assert fetch_bases(fasta=fasta_path, contig='chr2', length=3, start=1818) == 'AAC'
+        assert fetch_bases(fasta=fasta_path, contig='chr2', length=1, start=2030) == 'A'
+
+    def test_calculate_new_alleles(self):
```

Comment:
Suggestion to add some cases where ref and/or alt lenghts are greater than 1 to test the behaviour of complement vs. reverse complement as mentioned above.

Reply:
Or actually, did we decide that indels with reverse complemented alleles are out of scope for this specific PR? I forget. If this is the case, then please ignore for now :)

Reply:
I just added some cases for simple insertions and deletions